addon/components/mdc-tab-bar.js

```diff
@@ -12,6 +12,11 @@ const { strings } = MDCTabBarFoundation;
 export default Ember.Component.extend(MDCComponent, {
   //region Attributes
   /**
+   * @private
```

Comment:
The attribute `scrolling` only ever gets used inside `mdc-tab-bar-scroller.hbs`, where it gets passed into `{{mdc-tab-bar}}` as true. In that sense yes it's an attribute, but I marked it as private because this isn't something the user should ever be passing on their own into the `{{mdc-tab-bar}}`, it's handled internally by the code.